src/renderers/webgl/WebGLMultiview.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * @author fernandojsg / http://fernandojsg.com
+ * @author Takahiro https://github.com/takahirox
+ */
+
+import { WebGLMultiviewRenderTarget } from '../WebGLMultiviewRenderTarget.js';
+import { Matrix3 } from '../../math/Matrix3.js';
+import { Matrix4 } from '../../math/Matrix4.js';
+import { Vector2 } from '../../math/Vector2.js';
+
+function WebGLMultiview( renderer, gl, contextAttributes ) {
+
+	var DEFAULT_NUMVIEWS = 2;
+
+	var capabilities = renderer.capabilities;
+	var properties = renderer.properties;
+
+	var maxNumViews = capabilities.maxMultiviewViews;
+
+	var renderTarget, currentRenderTarget;
+	var mat3, mat4, cameraArray, renderSize;
+
+	function getMaxViews() {
+
+		return capabilities.maxMultiviewViews;
+
+	}
+
+	function getNumViews() {
+
+		if ( renderTarget && renderer.getRenderTarget() === renderTarget ) {
+
+			return renderTarget.numViews;
+
+		}
+
+		return 0;
+
+	}
+
+	function getCameraArray( camera ) {
+
+		if ( camera.isArrayCamera ) return camera.cameras;
+
+		cameraArray[ 0 ] = camera;
+
+		return cameraArray;
+
+	}
+
+	//
+
+	function isAvailable() {
+
+		return capabilities.multiview && ! contextAttributes.antialias;
```

Comment:
This seems like should go in `WebGLCapabilities`.

Reply:
done